### ISSUE TITLE
Networking

### ISSUE THREAD

**Issue by gheydon** [pducharme/UniFi-Video-Controller#48]
I have set up a new machine and installed Docker. I then installed unifi-video-controller image and started the container. The problem that I am getting is that unifi video is reporting that it is on a 172.x.x.x network instead of the 192.168.x.x network.

With this if I try to connect unifi video to my unifi login it then fails, and tries to connect to the 172.x.x.x network instead of to the docker host, and also it is not finding my camera that I have connect to my network (192.168.x.x) 

I am new to docker and I am pretty sure I do not have something set up in my network correctly.

I have done a lot of searching for information on this but it doesn't seem very popular to their software with docker.

Any help will be very much appreciated.

Gordon.

**Comment by chadhutchins182** [comment]
Can you post your docker run command (in code markdown please)?

**Comment by gheydon** [comment]
Hi, I just used the example command to start.

<code>
docker run         --name unifi-video         --cap-add SYS_ADMIN         --cap-add DAC_READ_SEARCH         -p 7442:7442         -p 7443:7443         -p 7445:7445         -p 7446:7446         -p 7447:7447         -p 7080:7080         -p 6666:6666         -v unifi-video-data:/var/lib/unifi-video         -v unifi-video-videos:/usr/lib/unifi-video/data/videos         -v unifi-video-logs:/var/log/unifi-video         -e PUID=99         -e PGID=100         -e DEBUG=1         pducharme/unifi-video-controller
</code>

Basically with docker I have installed it on a fresh machine, but I think I need to use the host network (which I have not had much success) or set up macvlan which will put the container on the local network.

Any help will be very much appreciated.
Gordon

**Comment by gheydon** [comment]
With using macvlan to set up a better bridge I was able to get it all working and be able to locate the cameras on my network.

The only thing I was not able to do is connect to the the video controller from video.ubnt.com. I am not 100% concerned about this right at the moment, but I will revisit this once I have the network set up and working with the security gateway pro instead of my old airport extreme I am currently running.

**Comment by fryfrog** [comment]
Inside Unifi Video, you just set your non-docker IP for each camera.

![screen shot 2018-02-26 at 5 09 20 pm](https://user-images.githubusercontent.com/87748/36704998-d0382416-1b17-11e8-9a8f-9364406d3415.png)

**Comment by fryfrog** [comment]
Since you've resolved this, I'm going to close the issue. :)
